pkg/v10/resource/chartoperator/create.go

```diff
@@ -37,7 +37,7 @@ func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createChange inte
 				resourcecanceledcontext.SetCanceled(ctx)
 
 				return nil
-			} else if helmclient.IsTillerInstallationFailed(err) {
+			} else if helmclient.IsTillerNotFound(err) {
```

Comment:
This is all renaming to use the error that is actually used. Except for reducing the backoff which I'll also highlight.